x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action.ts

```diff
@@ -255,42 +255,6 @@ export default ({ getService }: FtrProviderContext): void => {
       });
     });
 
-    it('should delete rules', async () => {
```

Comment:
Not sure I understand why these two have been deleted. Have they been moved somewhere else?

Reply:
Ah, ok, I see `should enable rules`  has been moved to `trial_license_complete_tier/perform_bulk_enable_disable.ts` and that Rule Deletion now has their own section.

My only question now is: for bulk rule deletion, the test that was in this file was under `Trial License`. The new `delete_rules_bulk.ts` file is under `Basic License`. Is that intended? Does it matter?

Reply:
Bulk enable/disable tests were moved to a separate file and enabled in MKI periodic/2nd quality gate pipelines since rule enabling/disabling functionality is critical.

Rule deletion test was removed by mistake instead of rule disabling test. Thanks for noticing it 👍